pkg/query/querier_test.go

```diff
@@ -43,7 +43,7 @@ func TestQueryableCreator_MaxResolution(t *testing.T) {
 	queryableCreator := NewQueryableCreator(nil, nil, testProxy, 2, 5*time.Second)
 
 	oneHourMillis := int64(1*time.Hour) / int64(time.Millisecond)
-	queryable := queryableCreator(false, nil, nil, oneHourMillis, false, false)
+	queryable := queryableCreator(false, nil, nil, oneHourMillis, false, false, TenantInfo{})
```

Comment:
```suggestion
	queryable := queryableCreator(false, nil, nil, oneHourMillis, false, false, nil)
```
See the above comment.